neutrons/SNAPRed
# Ewm792 reduction continue anyway wo norm or calib

## Description of work

This enables reduction to run in an uncalibrated mode.
This pr also reworks the error handling and pop-up logic a bit

to paste code blocks.
-->

## To test

run reduction with neither no calibration available
run it with no normalization available
run it with neither available,
see that it completes
```
For reduction/calib
Runnumber: 59039 

for calib:
sample: LA11b6
Grouping: Column
Peak Intensity Thresh: 0.01
Chi: 200
dmin: 1

for norm:
rnunumber: 58810
bgrunnumber: 58813
```



## Link to EWM item


[EWM#792](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=792)
